android/src/main/java/com/mybigday/rnmediameta/RNMediaMeta.java

```diff
@@ -78,7 +79,18 @@ private void putString(WritableMap map, String key, String value) {
     if (value != null) map.putString(key, value);
   }
 
-  private void getMetadata(String path, Promise promise) {
+  private void getMetadata(String pathOrJSONString, Promise promise) {
```

Comment:
Originally I just hope to replace the method, it doesn't break the JS function, we can still use  `get(path)` or `get(path, options`) to get a promise.

If we want to keep the origin native method for some reason, we can write a new native method like `getWithOptions`, then wrap the `NativeModules.RNMediaMeta`:

```
import { NativeModules, Platform } from 'react-native'

const { RNMediaMeta } = NativeModules;

export default {
  get(path, options) {
    if (options && Platform.OS === 'android') {
      return RNMediaMeta.getWithOptions(path, options)
    }
    return RNMediaMeta.get(path)
  }
}
```